core-blocks/archives/index.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ArchivesBlock from './block';
+export const name = 'core/archives';
+
+export const settings = {
+	title: __( 'Archives' ),
+
+	description: __( 'This block displays a monthly archive of your site’s Posts.' ),
+
+	icon: 'calendar-alt',
+
+	category: 'widgets',
+
+	keywords: [ __( 'archives' ) ],
```

Comment:
We could remove this line. Gutenberg will treat the title (Archives) as an implicit search keyword.